setup.py

```diff
@@ -18,7 +18,7 @@
     url='https://github.com/jarondl/pygtfs',
     packages=find_packages(),
     install_requires=['sqlalchemy>=0.7.8',
-                      'pytz>=2012d',
+                      'pytz>=2012',
```

Comment:
I ran into errors with any versions prior to `pytz 2014.9`. Would tou please bump this dependency to `'pytz>=2014.9',`?